### DISCUSSION TITLE
[Projects Beta] How to delete custom fields?

### DISCUSSION THREAD

**Discussion by creasty** [community/community#6460] [Projects and Issues] [answered]
Not sure how can we delete custom fields.

<img width="1112" alt="Screen Shot 2021-10-18 at 14 09 11" src="https://user-images.githubusercontent.com/1695538/137672528-bbb2d458-79dd-4a30-b3c6-441178f008cd.png">

**Comment by Nek-12** [comment]
The discussion was created like 9 days ago but still no answers and a single upvote. If some features are not available yet, there should be something like a "Current Limitations" document or discussion or whatever so that people know what to expect

**Comment by timoyates** [comment, marked as answer]
Hi folks, you can delete a field by opening its settings menu using the down chevron button to the right of its title: 
<img width="393" alt="Screenshot 2021-10-27 at 12 31 58" src="https://user-images.githubusercontent.com/6195786/139049512-5a5c5c91-738c-4a3d-b1a6-fad32efc1356.png">

**Reply by Nek-12** [reply]
Thanks for the reply! This is really confusing, because me and @creasty , I assume, were both expecting this option to be accessible while the field is hidden and/or from the field settings, at least, but both of these were blank. The problem is solved, but I guess this is a small design flaw nevertheless.

**Reply by timoyates** [reply]
Makes sense! I will share your feedback with the team 😊 Cheers!
